web/src/compose.js

```diff
@@ -169,7 +178,137 @@ export function send_message_success(request, data) {
     }
 }
 
-export function send_message(request = create_message_object()) {
+let split_long_messages_banner_shown = false;
+
+export function update_split_long_messages_info_banner() {
+    if (check_need_to_split_long_messages()) {
+        show_split_long_messages_info_banner();
+        compose_ui.autosize_textarea($("textarea#compose-textarea"));
+    } else {
+        clear_split_long_messages_info_banner();
+    }
+}
+
+const SPLIT_DELIMITER = "\n\n\n";
+/**
+ * checks whether the message actually needs to be split, as it is possible that `split_long_messages` is enabled but the message doesn't contain the delimiter.
+ */
+function check_need_to_split_long_messages() {
+    if (!is_split_long_messages_enabled()) {
+        return false;
+    }
+    const content = compose_state.message_content();
+    return content.includes(SPLIT_DELIMITER);
+}
+
+export function is_split_long_messages_enabled() {
+    return localStorage.getItem("split_long_messages") === "true";
+}
+
+function show_split_long_messages_info_banner() {
```

Comment:
Remove the word `long` from these function names; e.g., `show_split_messages_info_banner`. A common use case is to just do one message per 1-3 line bullet/paragraph, and that wouldn't be "long messages".